src/test/ts/core/YarnWorkspacesTest.ts

```diff
@@ -6,21 +6,21 @@ import * as YarnWorkspaces from '../../../main/ts/core/YarnWorkspaces';
 const assert = chai.use(chaiAsPromised).assert;
 
 describe('info', () => {
-  it('reads a test scenario with 2 modules', async () => {
-     assert.becomes(
-       YarnWorkspaces.info('src/test/data/yarninfo1'),
-       {
-         foo: {
-           location: 'foo',
-           workspaceDependencies: [],
-           mismatchedWorkspaceDependencies: []
-         },
-         bar: {
-           location: 'bar',
-           workspaceDependencies: [],
-           mismatchedWorkspaceDependencies: []
-         }
-       }
-     );
+  it('reads a test scenario with 2 modules', () => {
+    void assert.becomes(
```

Comment:
Why do you need the `void` keyword here? It is not getting returned anyway because of the `{}` around the function body, right?

Reply:
Lint error: dangling promise. But I should probably do it differently. Gimme a min.

Reply:
Ok. I've changed it to async..await.